gitignore

```diff
@@ -23,5 +24,10 @@ coverage
 build/Release
 
 # Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
```

Comment:
I think these folders are only created in your home folder (not the project folder) so I don't think they need to be in the `.gitignore` file. Did you add them because the folders showed up in the project folder?